samza-core/src/main/java/org/apache/samza/table/BaseReadWriteTable.java

```diff
@@ -66,4 +67,12 @@ public void init(Context context) {
   public String getTableId() {
     return tableId;
   }
+
+  public interface Func0 {
+    void apply();
```

Comment:
Can you take in a `Function` instead of defining our own `Func0`?

Reply:
The closest one is Runnable, which feels odd. Do you have a suggestion?

Reply:
>> The closest one is Runnable, which feels odd

I can also see why you think Runnable can be odd - esp., given its association with the Thread class.

>> Do you have a suggestion?

As a personal preference, I would use a `Runnable` instead of defining a Samza-specific interface with the same signature. I'll leave the final call to you.

Reply:
Yeah, I have a weird feeling about Runnable, so let's keep Func0 for now.